modules/imaging_uploader/jsx/columnFormatter.js

```diff
@@ -78,6 +78,28 @@ function formatColumn(column, cell, rowData, rowHeaders) {
     }
   }
 
+  if (column === 'Number Of MincCreated') {
+    let violatedScans;
+    if (row['Number Of MincCreated'] - row['Number Of MincInserted'] > 0) {
+      let numViolatedScans =
+           row['Number Of MincCreated'] - row['Number Of MincInserted'];
+      let uploadLoc = row.UploadLocation;
+      let fileName = uploadLoc.split('/').splice(-1)[0];
+      let patientName = fileName.split('.')[0];
```

Comment:
@davidblader 
There are situations where the filename is allowed a suffix at the end (with an `_`), such as for example Filename_2.zip, in that case, would the `_2` be passed on with the argument to the filter of the violated scans module? if so, this will not work as the patient name would still be withOUT the suffix at the end.